### PR TITLE
Implement toString methods in DTOs (now with Jackson)

### PR DESCRIPTION
**Pull request by BlackRose01**
Implement a toString-method to all DTOs. Already implemented methods were overwritten to be uniform. All toString-methods uses the Jackson ObjectMapper. Now all objects will return a JSON-compliant string.

Origin Pull Request: https://github.com/arrowhead-f/core-java-spring/pull/259

### CODE REVIEW

File: core-common/src/main/java/eu/arrowhead/common/dto/internal/AuthorizationInterCloudCheckRequestDTO.java [hunk 1]
```diff
@@ -14,49 +14,71 @@
...
-	
-	//-------------------------------------------------------------------------------------------------
-	public AuthorizationInterCloudCheckRequestDTO(final CloudRequestDTO cloud, final String serviceDefinition, final List<IdIdListDTO> providerIdsWithInterfaceIds) {
-		this.cloud = cloud;
-		this.serviceDefinition = serviceDefinition;
-		this.providerIdsWithInterfaceIds = providerIdsWithInterfaceIds;
-	}
-
-	//-------------------------------------------------------------------------------------------------
-	public CloudRequestDTO getCloud() { return cloud; }
```
Comment (borditamas) [inline core-common/src/main/java/eu/arrowhead/common/dto/internal/AuthorizationInterCloudCheckRequestDTO.java:33]: Why these formattings has been chaned in every dto class? We would expect only toString being added and not to change the whole class.

Comment (BlackRose01) [reply core-common/src/main/java/eu/arrowhead/common/dto/internal/AuthorizationInterCloudCheckRequestDTO.java:33]: It looks like that my IDE auto formattet all DTO classes. I only added the toString() method. When it is a problem then I will try to undo it.

Comment (borditamas) [reply core-common/src/main/java/eu/arrowhead/common/dto/internal/AuthorizationInterCloudCheckRequestDTO.java:33]: Please try to undo it.

**Comment by BlackRose01** [comment]
Done. But unfortunately I had to create a new pull request for it with the number #304

**Pull request closed without merging**